operator/src/test/java/oracle/kubernetes/operator/DomainStatusUpdateTestBase.java

```diff
@@ -1644,6 +1664,50 @@ void whenAdminOnlyAndAManagedServersShuttingDown_completedIsFalse() {
     assertThat(getRecordedDomain(), hasCondition(COMPLETED).withStatus(FALSE));
   }
 
+  @Test
+  void whenServerStateInfoNotConstructed_duringDomainResourceValidation_verifyDomainStatusNotUpdated() {
+    configureDomain().configureCluster(info, "cluster1").withReplicas(2).withMaxUnavailable(1);
```

Comment:
There is no need for the first underscore. Just change "_d" to "D"

If you change the MakeRight and ProcessingConstants to have the new behavior describe the effect on the Make-Right rather than who constructed it, you don't need both of these tests, as they do the same thing.